spec/features/generic_file_new_form_spec.rb

```diff
@@ -1,88 +1,74 @@
 require 'spec_helper'
-describe 'generic file new', :type => :feature do
-  let(:user) { FactoryGirl.create :user }
-  let!(:community1) do
```

Comment:
Moved all this data setup to where its actually needed. Only one test out of the 4 was using them.